scripts/uniffi_bindgen_generate_kotlin.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+BINDINGS_DIR="bindings/kotlin"
+TARGET_DIR="target/bindings/kotlin"
+PROJECT_DIR="ldk-node-jvm"
+PACKAGE_DIR="org/lightningdevkit/ldknode"
+UNIFFI_BINDGEN_BIN="cargo run --features=uniffi/cli --bin uniffi-bindgen"
+
+#rustup target add aarch64-apple-darwin
+#cargo build --target aarch64-apple-darwin || exit 1
+cargo build --release || exit 1
+$UNIFFI_BINDGEN_BIN generate bindings/ldk_node.udl --language kotlin -o "$TARGET_DIR" || exit 1
```

Comment:
I think we may need update this command here to point to the `bindings` folder one level up

```suggestion
$UNIFFI_BINDGEN_BIN generate ../bindings/ldk_node.udl --language kotlin -o "$TARGET_DIR" || exit 1
```